sys/net/application_layer/gcoap/gcoap.c

```diff
@@ -1266,7 +1267,7 @@ int gcoap_obs_init(coap_pkt_t *pdu, uint8_t *buf, size_t len,
     if (hdrlen > 0) {
         coap_pkt_init(pdu, buf, len, hdrlen);
 
-        uint32_t now       = xtimer_now_usec();
+        uint32_t now       = ztimer_now(ZTIMER_USEC);
         pdu->observe_value = (now >> GCOAP_OBS_TICK_EXPONENT) & 0xFFFFFF;
```

Comment:
I'm wondering if we could get rid of the `ZTIMER_USEC` dependency here, if RTT is used and running at 32Khz, then the base timer tick would be 30us. This brings the need for this feature to be exposed again though... So I guess that would be a follow-up, but in that case `ztimer_usec` could be avoided altogether for gcoap..

Reply:
Definetly another pr (maybe we even do not need that see chrysn 2021-11-17)

also: be careful with increasing frequencys they may lead to needing extra module (ztimer_extended) additional wakeups (for that extended)
i am also not sure if we consider ztimers bases open for non ztimer use

Digest: lets get it to ztimer and later improve its ztimer use.

Reply:
> Definetly another pr (maybe we even do not need that see chrysn 2021-11-17)

I think you forgot the link here :)

Reply:
https://github.com/RIOT-OS/RIOT/pull/17141#issuecomment-971740161